trello_to_pdf.py

```diff
@@ -125,6 +126,15 @@ async def has_error(page, card) -> bool:
             pass
     return False
 
+def collect_trello_card_hashes(directory):
+    card_hashes = set()  # Use a set to avoid duplicates
+    for root, dirs, files in os.walk(directory):
+        for file in files:
+            parts = file.split("_")
+            if parts:  # Check if the filename actually contains an underscore
+                card_hash = parts[0]
+                card_hashes.add(card_hash)
+    return card_hashes
```

Comment:
`nit`
```suggestion
def collect_trello_card_hashes(directory):
    return {file.split("_")[0] for root, dirs, files in os.walk(directory) for file in files if file.split("_")}
```

This would make it more Pythonic if you want to 😅

Reply:
It's a set comprehension ☺️